public/css/common.less

```diff
@@ -147,22 +151,12 @@ div.show-more {
     label {
       margin-right: 0;
     }
-
-    // Icinga Web 2 override
-    .spinner {
-      height: 100%;
-      padding-right: 0;
-    }
```

Comment:
I keep seeing this (but can't find the other occurrence now), but you don't re-add it somewhere else. What makes you think this isn't required anymore?

Reply:
I put it back.

Reply:
✅